pydantic_ai_slim/pydantic_ai/direct.py

```diff
@@ -273,9 +275,7 @@ class StreamedResponseSync:
     """
 
     _async_stream_cm: AbstractAsyncContextManager[StreamedResponse]
-    _queue: queue.Queue[messages.ModelResponseStreamEvent | Exception | None] = field(
```

Comment:
Subtle slightly-breaking change: this now also yields `FinalResultEvent` as seen in the docstring example above